js/components/dial.js

```diff
@@ -1494,29 +1496,33 @@ var DT_dial = (function () {
     } else {
       me.min = choose(me.block.min, -10);
       me.max = choose(me.block.max, 10);
```

Comment:
Can you set max to 10000 (now it's in Watt, previously kW)

Reply:
Done